ks/azure/centos76-hpc-sriov.ks

```diff
@@ -0,0 +1,401 @@
+
+# Deploy new configuration
+cat <<EOF > /etc/pam.d/system-auth-ac
+
+auth        required      pam_env.so
+auth        sufficient    pam_fprintd.so
+auth        sufficient    pam_unix.so nullok try_first_pass
+auth        requisite     pam_succeed_if.so uid >= 1000 quiet_success
+auth        required      pam_deny.so
+
+account     required      pam_unix.so
+account     sufficient    pam_localuser.so
+account     sufficient    pam_succeed_if.so uid < 1000 quiet
+account     required      pam_permit.so
+
+password    requisite     pam_pwquality.so try_first_pass local_users_only retry=3 authtok_type= ucredit=-1 lcredit=-1 dcredit=-1 ocredit=-1
+password    sufficient    pam_unix.so sha512 shadow nullok try_first_pass use_authtok remember=5
+password    required      pam_deny.so
+
+session     optional      pam_keyinit.so revoke
+session     required      pam_limits.so
+-session     optional      pam_systemd.so
+session     [success=1 default=ignore] pam_succeed_if.so service in crond quiet use_uid
+session     required      pam_unix.so
+
+EOF
+
+# Disable persistent net rules
+ln -s /dev/null /etc/udev/rules.d/75-persistent-net-generator.rules
+rm -f /lib/udev/rules.d/75-persistent-net-generator.rules /etc/udev/rules.d/70-persistent-net.rules 2>/dev/null
+rm -f /etc/udev/rules.d/70* 2>/dev/null
+ln -s /dev/null /etc/udev/rules.d/80-net-name-slot.rules
+
+# Disable NetworkManager handling of the SRIOV interfaces
+cat <<EOF > /etc/udev/rules.d/68-azure-sriov-nm-unmanaged.rules
+
+# Accelerated Networking on Azure exposes a new SRIOV interface to the VM.
+# This interface is transparently bonded to the synthetic interface,
+# so NetworkManager should just ignore any SRIOV interfaces.
+SUBSYSTEM=="net", DRIVERS=="hv_pci", ACTION=="add", ENV{NM_UNMANAGED}="1"
+
+EOF
+
+# Disable some unneeded services by default (administrators can re-enable if desired)
+systemctl disable wpa_supplicant
+systemctl disable abrtd
+systemctl disable firewalld
+
+# Update memory limits
+cat << EOF >> /etc/security/limits.conf
+*               hard    memlock         unlimited
+*               soft    memlock         unlimited
+*               soft    nofile          65535
+*               soft    nofile          65535
+EOF
+
+# Disable GSS proxy
+sed -i 's/GSS_USE_PROXY="yes"/GSS_USE_PROXY="no"/g' /etc/sysconfig/nfs
+
+# Enable reclaim mode
+echo "vm.zone_reclaim_mode = 1" >> /etc/sysctl.conf
+sysctl -p
+
+# Install Mellanox OFED
+mkdir -p /tmp/mlnxofed
+cd /tmp/mlnxofed
+wget http://www.mellanox.com/downloads/ofed/MLNX_OFED-4.5-1.0.1.0/MLNX_OFED_LINUX-4.5-1.0.1.0-rhel7.6-x86_64.tgz
+tar zxvf MLNX_OFED_LINUX-4.5-1.0.1.0-rhel7.6-x86_64.tgz
+
+KERNEL=$(uname -r)
+./MLNX_OFED_LINUX-4.5-1.0.1.0-rhel7.6-x86_64/mlnxofedinstall --kernel-sources /usr/src/kernels/$KERNEL --add-kernel-support --skip-repo
+
+sed -i 's/LOAD_EIPOIB=no/LOAD_EIPOIB=yes/g' /etc/infiniband/openib.conf
+/etc/init.d/openibd restart
+cd && rm -rf /tmp/mlnxofed
+
+# Install WALinuxAgent
+mkdir -p /tmp/wala
```

Comment:
Should we 'cd' to this directory after creating it?

Reply:
yep, thanks for catching this!